### PR TITLE
Add wait_for_completion to API documentation

### LINKED ISSUE #1312: [DOC] Update By Query API docs needs to be updated with `wait_for_completion` parameter
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
Update By Query API when passed with the parameter `wait_for_completion = false` returns a task id, which can identify async running tasks and later can be used to kill them to reclaim space.

Elasticsearch has documented this [here](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/docs-update-by-query.html#docs-update-by-query-task-api) and we have not.

Update By Query API docs in 
[OpenSearch](https://opensearch.org/docs/latest/opensearch/rest-api/document-apis/update-by-query)
[Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/docs-update-by-query.html#docs-update-by-query-task-api) 
 
Below is an example:

Request
```
POST test_index/_update_by_query?wait_for_completion=false
{
  "query": { 
    "term": {
      "field": "test"
    }
  }
}
```

Response
```
{
  "task" : "task_id"
}
```

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
Tested this in Opensearch 2.2.1, but this was available in ES long before the fork, so I believe this is available in all OS versions.

Screen_Shot_2022-09-23_at_2_12_22_PM

### PR DESCRIPTION
**Pull request by ariamarble**
### Description

Adds wait_for_completion URL parameter to API documentation

### Issues Resolved

#1312 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/rest-api/document-apis/update-by-query.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ terminate_after | Integer | The maximum number of documents OpenSearch should pr
 timeout | Time | How long the operation should wait from a response from active shards. Default is `1m`.
 version | Boolean | Whether to include the document version as a match.
 wait_for_active_shards | String | The number of shards that must be active before OpenSearch executes the operation. Valid values are `all` or any integer up to the total number of shards in the index. Default is 1, which is the primary shard.
+wait_for_completion | boolean | When set to `false` a task ID is returned which can identify the async tasks that are running. You can then use the task ID to kill the task and reclaim space. Defaults to `true`.
```
Comment (Naarcha-AWS) [inline _opensearch/rest-api/document-apis/update-by-query.md:80]: Let's add a comma after `false`. 

```suggestion
wait_for_completion | boolean | When set to `false`, a task ID is returned which can identify the async tasks that are running. You can then use the task ID to kill the task and reclaim space. Defaults to `true`.
```

**Review by Naarcha-AWS — APPROVED** [review]
One small suggestion, otherwise LGTM.

File: _opensearch/rest-api/document-apis/update-by-query.md (hunk 1 above)
Comment (ariamarble) [reply _opensearch/rest-api/document-apis/update-by-query.md]: done

Comment (hdhalter) [reply _opensearch/rest-api/document-apis/update-by-query.md]: Did Kiran sign off on this? [awskiran@amazon.com](mailto:awskiran@amazon.com)

Comment (ariamarble) [reply _opensearch/rest-api/document-apis/update-by-query.md]: yeah, working on some changes

**Comment by ariamarble** [comment]
Made additional changes that are signed off by the developer

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Pull request merged**